packages/cubejs-playground/src/events.ts

```diff
@@ -8,18 +8,23 @@ let baseProps = {
   sentFrom: 'frontend',
 };
 let telemetry: boolean | undefined;
+let track: null | ((event: Record<string, any>, telemetry?: boolean) => Promise<any>) = null;
```

Comment:
Maybe it's better to assign a default implementation to protect a case when someone is using the playground as the library and it will be a BC for him.